rm pdfs revise existing docs (#213)
digital-asset/cn-quickstart: sdk/docs/sharable/sdk/quickstart/observe/observability-troubleshooting-overview.rst

Grafana
~~~~~~~

> The Grafana interface is accessible via its web interface which is port-mapped to http://localhost:3030/, 

Reviewer: ```suggestion
Grafana is accessible via its web interface, which is port-mapped to http://localhost:3030/, 
```

and can be opened in the current browser from the command line using ``make open-observe``.

> It is recommended that the focus of your debugging should be on using the trace and log facilities provided by Grafana and ledger inspection using Daml Shell. 

Reviewer: ```suggestion
Your debugging should focus on using Grafana's trace and log facilities, as well as ledger inspection via Daml Shell. 
```

> Ensuring that the exported logs and traces are sufficient to support debugging during development also provides assurance that they will be sufficient to support diagnostics in production.

Reviewer: ```suggestion
If you make sure that your exported logs and traces are sufficient to support debugging during development, they are more likely to support diagnostics in production as well.
```

> There is additional access configured into the quickstart that can assist with debugging on ``LocalNet``. 

Reviewer: ```suggestion
There is additional access configured into the Quickstart that can assist with debugging on ``LocalNet``. 
```

      - "5055:5005"

> This enables remote debugging of the java component backend in the user application (backend-service). 

Reviewer: ```suggestion
This enables remote debugging of the Java component backend in the user application (backend-service). 
```

You can use this to connect an IDE Debugger to the service at runtime if required. 

> Keep in mind that we recommend your first resort be Grafana and the consolidated logs in Loki. 
> This keeps the system debugable in production.

Reviewer: ```suggestion
We recommend Grafana as your first resort, along with the consolidated logs in Loki. 
This keeps the system debuggable in production.
```

Observability and tracing
=========================

Faulty distributed systems can be notoriously hard to diagnose.

> Quickstart provides, at the start of a project, 
> the sort of observability and diagnostics facilities often only developed toward the end of the project. 
> Simplifying diagnostics for new Canton Network Applications from the outset of each project is one of the motivations behind the development of Quickstart.

Reviewer: ```suggestion
From the start of a project, Quickstart provides the sort of observability and diagnostics facilities that are otherwise often only developed toward the end. 
Simplifying diagnostics for new Canton Network Applications from the outset of each project is one of the motivations behind the development of Quickstart.
```

The links in the overview include the official user and reference documentation for the various tools included in Quickstart. 
While there is no substitute for the official documentation, 
it is hoped the following tour of the capabilities configured into Quickstart can provide a starting point for your own experimentation.

Correlation identifiers
-----------------------

> Inspecting any distributed system invariably begins by correlating identifiers—Canton is no different in that regard. 
> Canton can accept and/or generate a number of identifiers suitable for correlating across both time, various nodes, and the evolving state of the ledger.

Reviewer: ```suggestion
Inspecting Canton begins by correlating identifiers, much like inspecting any other distributed system. 
Canton can accept and/or generate a number of identifiers suitable for correlating across both time, various nodes, and the evolving state of the ledger.
```

     -   ``Global, potentially non-unique, identifier for a legal entity on the Canton ledger.`` [8]_

The goal of the observability configuration is to make it easier to navigate through the provenance of any state or event in the wider system. 

> Any or all of these identifiers can be used to correlate a combination of logs, metrics, state. 

Reviewer: ```suggestion
Any or all of these identifiers can be used to correlate a combination of logs, metrics, and state. 
```
